### ISSUE TITLE
stack build systematic hang (-j2 on Mac)

### ISSUE THREAD

**Issue by JacquesCarette** [JacquesCarette/Drasil#3139]
I don't know if this is known upstream, but I do know that on my laptop, it just hangs randomly.  Looks like -j1 does not. It is really quite annoying, but at least now I know how to get around it.

So this is more of an investigation ticket.

**Comment by balacij** [comment]
By any chance, are you using the M* arm version? I vaguely remember some chatter on Twitter about GHC being buggy with 8.8.4 on  M*. If so, it might be worth it for us to look into creating a few extra stack profiles to use newer GHC versions.

**Comment by JacquesCarette** [comment]
Nope, I'm on a very old Mac. And I think I'm on 9.2, though I'm not sure (stack makes that a little hard to tell).

**Comment by balacij** [comment]
Well, then, this issue is surprising.

By the way, if you're using the Makefile, it should be using the basic `stack.yaml`s stackage version, so it should be tied GHC 8.8.4 (Stackage LTS 16.31).

**Comment by balacij** [comment]
Out of curiosity, does this still happen?

**Comment by JacquesCarette** [comment]
To be honest, I haven't tried in a while. I'm sure I'll get back to doing Drasil dev soon, and I'll see one way or another then.
